scss/6-components/_contact_details.scss

```diff
@@ -4,4 +4,8 @@
   p {
     margin-bottom: 0;
   }
+
+  p:first-child {
```

Comment:
This feels like it's asking for trouble. What happens if a the HTML doesn't start with a `<p>`. We have have no control over that right?

Reply:
I agree, the issue we have currently is that the content model does not support it as the content being migrated has no concept of a title, it is just a blob. Perhaps this is something to be resolved by adding a title field (a proper one, there is one now but that is hijacked by the migration script just putting the episerver id) and the migration script fixing migrated content. But that is a whole ticket in itself. It is an issue with other things, eg titles in Callouts.